docker-compose.yml

```diff
@@ -0,0 +1,27 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:13.1-alpine
+    volumes:
+      - postgres_data:/var/lib/postgresql/data
+    environment:
+      - POSTGRES_USER=smallboard_user
+      - POSTGRES_PASSWORD=smallboard_password
+      - POSTGRES_DB=smallboard
+  web:
+    build: .
+    command: python manage.py runserver 0.0.0.0:8000
+    volumes:
+      - .:/usr/src/smallboard/:Z
+    ports:
+      - 8000:8000
+    environment:
+      - DATABASE_URL=postgres://smallboard_user:smallboard_password@db/smallboard
+    env_file:
+      - ./.env
```

Comment:
What do you think of just removing this `env_file` entirely? Now that we configure `DATABASE_URL` in `environment` the only other setting is `ACTIVE_HUNT_ID`, which will be deprecated by https://github.com/cardinalitypuzzles/smallboard/pull/201

Maybe more importantly, it reduces the possibility of configuring env vars in two separate places. For `docker-compose` it might make more sense to just use `environment` and complete ignore the `env_file`. Any local tweaks to the environment can be made in the `docker-compose.yaml` file.

Reply:
Loading some environment variables from an env_file might still be nice because there are some environment variables that we don't want to check in, like the GOOGLE_API_PRIVATE_KEY and later, the Discord API key.

Reply:
Right, I think we still want this for all our secrets and stuff.